From 51cafcc92cc9c10cdf7b27ba052459adfb766a83 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Fri, 4 Mar 2016 18:18:37 +0800
Subject: [PATCH 0973/5242] MLK-12478-2 dts: imx7d-lpddr3-arm2: add lpsr mode
 state for flexcan pins

commit  fbb5bd6c3062e9fb9461a0528ff383495cc5e3e4 from
https://source.codeaurora.org/external/imx/linux-imx.git

add lpsr mode state for flexcan pins

Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index b177cdb..eb87225 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -228,15 +228,17 @@
 };
 
 &flexcan1 {
-	pinctrl-names = "default";
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_flexcan1>;
+	pinctrl-1 = <&pinctrl_flexcan1>;
 	xceiver-supply = <&reg_can1_3v3>;
 	status = "disabled";
 };
 
 &flexcan2 {
-	pinctrl-names = "default";
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_flexcan2>;
+	pinctrl-1 = <&pinctrl_flexcan2>;
 	xceiver-supply = <&reg_can2_3v3>;
 	status = "disabled";
 };
-- 
1.7.9.5

